case/src/test/kotlin/com/ritense/case/web/rest/CaseDefinitionResourceIntTest.kt

```diff
@@ -78,11 +84,13 @@ class CaseDefinitionResourceIntTest: BaseIntegrationTest() {
     @Test
     fun `should update case settings`() {
 
-        documentDefinitionService.deploy("" +
-            "{\n" +
-            "    \"\$id\": \"resource-test-update.schema\",\n" +
-            "    \"\$schema\": \"http://json-schema.org/draft-07/schema#\"\n" +
-            "}\n")
+        documentDefinitionService.deploy(
+            "" +
```

Comment:
with the way it's formatted now you don't need the empty string. same goes for the one below
.